docs/source/user/api_change.rst

```diff
@@ -57,6 +67,8 @@ Added in OpenFAST v2.6.0
 Module                                        Line  Flag Name        Example Value
 ============================================= ==== =============== ========================================================================================================================================================================================================
 AeroDyn 15                                         TwrTi               0.0000000E+00  6.0000000E+00  1.0000000E+00  1.0000000E-01                 [additional column in *Tower Influence and Aerodynamics* table]
+AeroDyn 15                                    40   UAStartRad          0.25    UAStartRad          - Starting radius for dynamic stall (fraction of rotor radius) [used only when AFAeroMod=2; if line is missing UAStartRad=0]
```

Comment:
Was this change introduced in 2.6, or with this PR (in which case this and the next line should go with the section above).

Reply:
Figured this out.  This development branch existed prior to v2.6 and v3.0 releases.  Resolved with commit.